feature-group-definitions/input-event.yml

```diff
@@ -0,0 +1,4 @@
+spec: https://w3c.github.io/uievents/#event-type-input
+caniuse: input-event
+compat_features:
+  - api.HTMLElement.input_event
```

Comment:
I think we should also include these features from https://developer.mozilla.org/en-US/docs/Web/API/InputEvent#browser_compatibility:

api.InputEvent
api.InputEvent.InputEvent
api.InputEvent.data
api.InputEvent.dataTransfer
api.InputEvent.getTargetRanges
api.InputEvent.inputType
api.InputEvent.isComposing

(But not api.InputEvent.inputType.insertFromPasteAsQuotation.)